server/jamovi/core.pyx

```diff
@@ -291,18 +300,21 @@ cdef class Column:
         else:
             self._this.append[int](value)
 
-    def append_level(self, raw, label):
-        self._this.appendLevel(raw, label.encode('utf-8'))
+    def append_level(self, raw, label, importValue):
+        self._this.appendLevel(raw, label.encode('utf-8'), importValue.encode('utf-8'))
 
-    def insert_level(self, raw, label):
-        self._this.insertLevel(raw, label.encode('utf-8'))
+    def insert_level(self, raw, label, importValue):
+        self._this.insertLevel(raw, label.encode('utf-8'), importValue.encode('utf-8'))
```

Comment:
if you do this:

```
    def insert_level(self, raw, label, importValue=None):
        if importValue is None:
            importValue = label
        self._this.insertLevel(raw, label.encode('utf-8'), importValue.encode('utf-8'))
```

it will simplify a lot of your code further down.